tests/integration/individual_response/test_individual_response.py

```diff
@@ -1,14 +1,21 @@
-import re
+from datetime import datetime
 from unittest.mock import MagicMock
 
+from dateutil.tz import tzutc
+from freezegun import freeze_time
+
 from app import settings
 from app.publisher.exceptions import PublicationFailed
 from tests.integration.integration_test_case import IntegrationTestCase
 
 
+@freeze_time("2020-11-25T11:59:00")
 class IndividualResponseTestCase(IntegrationTestCase):
     def setUp(self):
         settings.EQ_INDIVIDUAL_RESPONSE_LIMIT = 2
+        settings.EQ_INDIVIDUAL_RESPONSE_POSTAL_DEADLINE = datetime.fromisoformat(
+            "2020-11-25T12:00:00"
+        ).astimezone(tz=tzutc())
```

Comment:
Based on Andrew's comment, I think it worth converting all `tz=tzutc()` to `tz=timezone.utc` for the files you have changed just for consistency. We can progress a card to update rest of runner to ensure we are consistent.